.github/workflows/ubuntu.yml

```diff
@@ -0,0 +1,31 @@
+name: Ubuntu
+
+on: [pull_request, workflow_dispatch]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  gnu:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        build_type: [Debug, Release]
+    env:
+      FC: gfortran
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

Comment:
checkout@v4 is the most recent version now.

Reply:
done